core/src/Kokkos_Atomics_Desul_Config.hpp

```diff
@@ -23,10 +23,6 @@ static_assert(false,
 
 #include <Kokkos_Macros.hpp>
 
-#ifdef KOKKOS_ENABLE_OPENMPTARGET
-#define DESUL_HAVE_OPENMP_ATOMICS
-#endif
```

Comment:
`CUDA-11.7-NVCC` build that uses generated makefiles fails
I forgot about non-cmake builds...